### ISSUE TITLE
We should add `provider` as a label to each metric so we can show where our data is coming from

### ISSUE THREAD

**Issue by etsauer** [redhat-cop/pelorus#206]

**Comment by KevinMGranger** [comment]
Related to #225

**Comment by mpryc** [comment]
@etsauer What would be the value of this `provider` label? What is the use case for this metric?

**Comment by etsauer** [comment]
@mpryc no, I was thinking more generic than that, like 'github', 'gitlab', 'jira'.. basically the name of the datasource where the information came from. I see this being useful in two ways:

1. to help the dashboard de-mystify how pelorus actually works
2. to help debug in case some users might have multiple providers.
